examples/Gesture.py

```diff
@@ -0,0 +1,40 @@
+"""
+Example program for receiving gesture events and accelerometer readings from Kai
+"""
+
+import os
+import time
+
+from KaiSDK.WebSocketModule import WebSocketModule
+from KaiSDK.DataTypes import KaiCapabilities
+import KaiSDK.Events as Events
+
+# Event listener functions
+def gestureEvent(ev):
+    print(ev.gesture)
+
+def accelerometerEv(ev):
+    print(ev.accelerometer.x)
+
+moduleID = os.environ.get("MODULE_ID")
```

Comment:
Why are you getting this from an environment variable?
Not like it's a bad thing, but why not from a config file that's gitignored or something? Cuz people will take this as an example and follow suit

Reply:
It was an env variable on my system, just used the same thing. Doesn't make a big diff, but i changed it to a config file and mentioned a comment asking the user to use their module's ID and secret